src/core/CoolBoard.cpp

```diff
@@ -286,23 +295,12 @@ bool CoolBoard::config() {
   config.set<bool>(json, "sleepActive", this->sleepActive);
   config.set<bool>(json, "manual", this->manual);
   config.set<String>(json, "mqttServer", this->mqttServer);
-  this->mqttClient.setClient(this->wifiClient);
-  this->mqttClient.setServer(this->mqttServer.c_str(), 1883);
-  this->mqttClient.setCallback(
-      [this](char *topic, byte *payload, unsigned int length) {
-        this->mqttCallback(topic, payload, length);
-      });
-  this->mqttId = WiFi.macAddress();
-  this->mqttId.replace(F(":"), F(""));
-  this->mqttOutTopic =
-      String(F("$aws/things/")) + this->mqttId + String(F("/shadow/update"));
-  this->mqttInTopic = String(F("$aws/things/")) + this->mqttId +
-                      String(F("/shadow/update/delta"));
-  if (!config.writeJsonToFile()) {
+  this->mqttsConfig();  
+  /*if (!config.writeJsonToFile()) {
```

Comment:
No way ! **never** commit commented out code! If this code is now useless, it needs to go away for good!

Reply:
And I really think this code should stay for the moment.